conans/client/cmd/new.py

```diff
@@ -91,11 +95,15 @@ class {package_name}Conan(ConanFile):
     description = "<Description of {package_name} here>"
     topics = ("<Put some tag here>", "<here>", "<and here>")
     settings = "os", "compiler", "build_type", "arch"
-    options = {{"shared": [True, False]}}
-    default_options = {{"shared": False}}
+    options = {{"shared": [True, False], "fPIC": [True, False]}}
+    default_options = {{"shared": False, "fPIC": True}}
```

Comment:
Seems a little odd to have compilation settings if it's header only 🤔 

base on the example CCI points to https://github.com/conan-io/conan-center-index/blob/78363b7d99786ea0060bfe2a02ded90b43bf2d4a/recipes/rapidjson/all/conanfile.py

we should have

```suggestion
    no_copy_source = True
```

Reply:
I think the `conanfile_header` temple below addresses header_only packages.
This one addresses building from current folder (=conanfile bundled with sources).